modules/conductor/src/main/java/org/opencastproject/event/handler/AssetManagerUpdatedEventHandler.java

```diff
@@ -265,6 +269,16 @@ public void handleEvent(final SeriesItem seriesItem) {
           assetManager.takeSnapshot(snapshot.getOwner(), mp);
         } catch (AssetManagerException e) {
           logger.error("Error updating mediapackage {}", mp.getIdentifier().toString(), e);
+        } finally {
+          if (mpAclAttachmentTuple != null) {
+            try {
+              workspace.delete(mpAclAttachmentTuple.getB().getURI());
+            } catch (Exception ex) {
+              // We only want to clean up. If the file is gone, that is fine too.
+              logger.debug("Unable to delete series acl attachment for media package {} after updating it.",
```

Comment:
I agree with the log message. 

We can't use `IOUtils.deleteQuietly` because `mpAclAttachmentTuple` contains the URL to the file in workspace (e.g. https://…) and not a file path on local storage.